configs/icds-7lakh-jan2019.yml

```diff
@@ -0,0 +1,639 @@
+estimation_buffer: 0.2
+estimation_growth_factor: 0.01  # buffer increases by 1% per month
+storage_buffer: 0.33  # keep max storage at 75% of disk
+storage_display_unit: TB
+
+summary_dates:
+#  - '2018-10'
```

Comment:
Did you intentionally leave lines commented out?

Reply:
yes, intentionally commented out, this just changes what summaries are included in the output